docs/getting-started.md

```diff
@@ -115,18 +115,28 @@ $Version="1.6.4"
 curl.exe -L https://github.com/containerd/containerd/releases/download/v$Version/containerd-$Version-windows-amd64.tar.gz -o containerd-windows-amd64.tar.gz
 tar.exe xvf .\containerd-windows-amd64.tar.gz
 
-# Copy and configure
-Copy-Item -Path ".\bin" -Destination "$Env:ProgramFiles\containerd" -Recurse -Container:$false -Force
-cd $Env:ProgramFiles\containerd\
-.\containerd.exe config default | Out-File config.toml -Encoding ascii
+# If containerd previously installed run:
+Stop-Service containerd
+
+# Copy
+Copy-Item -Path .\bin\* -Destination (New-Item -Type Directory $Env:ProgramFiles\containerd -Force) -Recurse -Force
+
+# add the binaries (containerd.exe, ctr.exe) in $env:Path
+$Path = [Environment]::GetEnvironmentVariable("PATH", "Machine") + [IO.Path]::PathSeparator + "$Env:ProgramFiles\containerd"
+[Environment]::SetEnvironmentVariable( "Path", $Path, "Machine")
+# reload path, so you don't have to open a new PS terminal later if needed
+$Env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";" + [System.Environment]::GetEnvironmentVariable("Path","User")
+
+# configure
+containerd.exe config default | Out-File $Env:ProgramFiles\containerd\config.toml -Encoding ascii
 
 # Review the configuration. Depending on setup you may want to adjust:
 # - the sandbox_image (Kubernetes pause image)
 # - cni bin_dir and conf_dir locations
-Get-Content config.toml
+Get-Content $Env:ProgramFiles\containerd\config.toml
 
 # Register and start service
-.\containerd.exe --register-service
+containerd.exe --register-service
```

Comment:
containerd.exe logs go into the ether when we start containerd as a service using windows service manager. Could we suggest using nssm to configure logs to go into a cyclic buffer and starting service? 

For example: https://github.com/Azure/AgentBaker/blob/f13623d9f2683081d28adaf5607105f3f732f24f/staging/cse/windows/containerdfunc.ps1#L21

Reply:
Perhaps as a side note?

Reply:
sure! that would be helpful

Reply:
@kiashok -- pls take a look, I've added the tip. Thanks!